docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,60 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+= Creating tags
+
+You can create custom tags for applications that {ProductShortName} migrates.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
++
+image::mta-web-assessment-tags-01.png[Tags]
+
+. Click *Create tag*, opening a dialog.
+. In the *Name* field enter a unique name for the tag.
+. Click the *Tag category* field and select a category tag to belong.
+. Click *Create*.
+
+[id="editing-tags_{context}"]
+= Editing tags
+
+You can edit defined tags.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab. A list of tag categories appears in the main pane.
+. Click the arrow to the left of the category name, opening a list of tags in the category.
++
+image::mta-web-assessment-tags-03.png[Tags]
+
+. Select *Edit* from the drop-down menu, opening a dialog.
+. Edit the tag's name in the *Name* field.
+. Click the *Tag category* field and select which the tag category the tag belongs. 
+. Click *Save*.
+
+
+[id="editing-tag-categories_{context}"]
+= Editing tag categories
+
+You can edit defined tag categories.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
++
+image::mta-web-assessment-tags-01.png[Tags]
```

Comment:
I feel it is redundant because of the 2 first images. The user is probably already familiar with the Tags page

Reply:
Removed